src/components/molecules/SearchAutocomplete/suggestionsList.tsx

```diff
@@ -17,14 +19,23 @@ const SuggestionsList: FC<SuggestionsProps> = ({
       aria-labelledby="search-input-label"
```

Comment:
According to this article https://tink.uk/the-difference-between-aria-label-and-aria-labelledby/, we could use here just `aria-label`

Reply:
Right, I've changed that :)